.github/workflows/performance_monitoring.yml

```diff
@@ -27,7 +26,9 @@ jobs:
           rm -rf performance-monitoring
           git clone git@github.com:taichi-dev/performance-monitoring.git
           cd performance-monitoring
+          export CXX=clang
```

Comment:
nit: better put this in the `env` section below before merging, also `CXX` should point to clang++-10 ;)